filterList.php

```diff
@@ -47,28 +76,51 @@
         foreach($filtered_tasks as $task){
             echo "<br>Title:".$task['taskTitle']."<br>";
             echo "Description: ".$task['description']."<br>";
-    
+            echo "Importance: ".$task['importance']."<br>";
+            echo "Type of Work: ".$task['typeOfWork']."<br>";
+            echo "Due Date: ".gmdate("Y-m-d", $task['dueDate'])."<br>";
         }
     } //dealing with no result outputs(edge cases tested)
     else {
     echo "<br> No results found - please try again. <br>";
     }
-
+ 
 }
 ?>
 
 <!-- Form to take in user inputs -->
 <form action="filterList.php?id=<?php echo $_GET['id']; ?>" method="POST"> 
 
     <br>
-    <input type="text" name="filt_name" value="" class="form-control" placeholder="Task Title"/>
+    <input type="text" name="filt_name" value="" class="form-control" placeholder="Task Title" maxlength="255"/>
     <br>
-    <input type="number" name="filt_effort" value="" class="form-control" placeholder="Importance"/>
+    <input type="number" name="filt_effort" value="" class="form-control" placeholder="Importance" min = "0" max = "10" step="1"/>
     <br>
-    <input type="text" name="filt_work" value="" class="form-control" placeholder="Type of Work"/>
+    <input type="text" name="filt_work" value="" class="form-control" placeholder="Type of Work" maxlength="255"/>
```

Comment:
good addition to the inputs. Users should be constrained so there are no errors when making a query! 👍